### PR TITLE
fix: ApiManagement-Service - Changed `loggers` to `logger`

### PR DESCRIPTION
**Pull request by AlexanderSehr**
## Description

- Changing folder name to singular to align with AVM spec: [resource module naming](https://azure.github.io/Azure-Verified-Modules/specs/shared/#bicep-resource-module-naming)
- Fixed incorrect description. See failed test [here](https://github.com/Azure/bicep-registry-modules/actions/runs/11607584544)

## Pipeline Reference

| Pipeline |
| -------- |
|    [avm.res.api-management.service](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.api-management.service.yml)      |

## Type of Change

- [ ] Update to CI Environment or utilities (Non-module affecting changes)
- [ ] Azure Verified Module updates:
  - [x] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

### CODE REVIEW

File: avm/res/api-management/service/logger/README.md
Comment (AlexanderSehr) [inline avm/res/api-management/service/logger/README.md:1]: Please note, the `main.json` templates **must** be re-build with the latest Bicep version `0.31.34` as this version (running on the agents) renders a different `main.json` file then previous versions. 

The steps to execute would be
- Upgrade your Bicep CLI version to `0.31.34` (e.g. via `choco upgrade bicep` if choco is used)
- Re-run the `Set-AVMModule` script on your module. E.g. via `Set-AVMModule -ModuleFolderPath '.\avm\res\automation\automation-account\' -SkipFileAndFolderSetup -SkipReadMe -Recurse`
- Push the changes
- Re-run the tests to ensure everything works. 

If the static tests continue to fail for you, please note that there is currently presumably another new bug which affects some of our modules ([ref](https://github.com/Azure/bicep/issues/15517)).

Comment (AlexanderSehr) [reply avm/res/api-management/service/logger/README.md:1]: addressed

**Comment by AlexanderSehr** [comment]
Hey @tony-box, could you give this PR a spin some time? 💪

**Comment by tony-box** [comment]
> Hey @tony-box, could you give this PR a spin some time? 💪

Looks good, sir! Sorry for the delay. My dev environment is dead and I've been swamped with other work :(

Thank you for the update!

**Review by tony-box — COMMENTED** [review]
looks good

**Comment by AlexanderSehr** [comment]
> @Azure/avm-module-reviewers-bicep whoops, i accidentally closed this without merging. I reopened and I think it reset the reviews. Can someone please re-review?

Not onyl that, but it also removed you as a reviewer 😄 A feature, I guess.

**Review by krbar — APPROVED** [review]

**Pull request merged**